vspace/src/main/java/edu/asu/diging/vspace/web/staff/AddChoiceBlockController.java

```diff
@@ -25,21 +26,33 @@
     @Autowired
     private IContentBlockManager contentBlockManager;
 
+    @Autowired
+    private ContentBlockRepository contentBlockRepository;
```

Comment:
controllers should not directly call repositories (that's the job of the manager/service classes)